docs/reference/content/driver/tutorials/ssl.md

```diff
@@ -201,3 +201,49 @@ ERefGuide.html).
 Some applications may want to force only the TLS 1.2 protocol. To do this, set the `jdk.tls.client.protocols` system property to "TLSv1.2".
 
 Java runtime environments prior to Java 8 started to enable the TLS 1.2 protocol only in later updates, as shown in the previous section. For the driver to force the use of the TLS 1.2 protocol with a Java runtime environment prior to Java 8, ensure that the update has TLS 1.2 enabled.
+
+
+## OCSP
+
+{{% note %}}
+The Java driver cannot enable OCSP by default on a per MongoClient basis.
+{{% /note %}}
+
+### Client-driven OCSP
+
+An application will need to set JVM system and security properties to ensure that client-driven OCSP is enabled:
+
+-  `com.sun.net.ssl.checkRevocation`:
+      When set to `true`, this system property enables revocation checking.
+
+-  `ocsp.enable`:
+      When set to `true`, this security property enables client-driven OCSP.
+      
+To configure an application to use client-driven OCSP, the application must already be set up to connect to a server using TLS. Setting these system properties is required to enable client-driven OCSP.
+
+### OCSP Stapling
+
+{{% note class="important" %}}
+Red Hat Enterprise Linux 7 does not support OCSP stapling.
+{{% /note %}}
+
+{{% note class=important %}}
+The following exception may occur when using OCSP stapling with Java runtime environments that use the TLS 1.3 protocol (Java 11 and higher use TLS 1.3 by default):
+
+`javax.net.ssl.SSLHandshakeException: extension (5) should not be presented in certificate_request`
+
+The exception is due to an exception handling issue with TLS 1.3 in Java 11 and higher. To avoid this exception when using a Java runtime environments using the TLS 1.3 protocol, you can force the application to use the TLS 1.2 protocol. To do this, set the `jdk.tls.client.protocols` system property to "TLSv1.2".
+{{% /note %}}
+
+An application will need to set several JVM system properties to set up OCSP stapling:
```

Comment:
Nit: I believe "stapled OCSP" might be slightly more correct, since the application is consuming the server's stapled OCSP response and not stapling the revocation status of the client certificate.

Reply:
I now realize that the JSEE docs refer to stapled OCSP as "OCSP stapling" , so I leave it you as to which term you prefer :)